fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/Sequence.java

```diff
@@ -37,7 +38,12 @@ public void apply(Integer priorVersion, IDatabaseAdapter target) {
         if (priorVersion != null && priorVersion > 0 && this.version > priorVersion) {
             throw new UnsupportedOperationException("Upgrading sequences is not supported");
         }
-        apply(target);
+
+        // Only if VERSION1 then we want to apply, else fall through
+        // Re-creating a sequence can have unintended consequences.
+        if (this.version == 1 && priorVersion == 0) {
```

Comment:
```suggestion
        if (this.version == 1 && (priorVersion == 0 || priorVersion == null)) {
```

Reply:
I didn't run into the situation where it was null.  I had some logging statements, and it was always zero or 1

Reply:
Adding as it's a good guard (just in case)